### ISSUE TITLE
Error when using Zukini via nuget

### ISSUE THREAD

**Issue by s-kavanagh** [MikeMugu/Zukini#16]
Hi Mike, 

I've been trying to get to the bottom of an error and I've boiled it down as far as this.  If I clone the most recent Zukini repo straight from github and carry out the following steps:
1) Remove references to Zukini from Zukini.Examples.Pages and Zukini.Examples.Features.
2) Use the nuget manager and add Zukini to the 2 above projects.
3) Try running any test

I get the following error:

```
Result StackTrace:  
at Zukini.Hooks.BeforeScenario()
   at lambda_method(Closure , IContextManager )
   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.FireEvents(HookType bindingEvent)
   at TechTalk.SpecFlow.Infrastructure.TestExecutionEngine.OnScenarioStart(ScenarioInfo scenarioInfo)
   at TechTalk.SpecFlow.TestRunner.OnScenarioStart(ScenarioInfo scenarioInfo)
   at Zukini.Examples.Features.Features.SmokeTestFeature.ScenarioSetup(ScenarioInfo scenarioInfo)
   at Zukini.Examples.Features.Features.SmokeTestFeature.PerformAGoogleSearchForSpecFlowReturnsSpecflow_OrgSite() in C:\Code\Zukini\Zukini.Examples.Features\Features\SmokeTest.feature:line 7
Result Message: System.MissingMethodException : Method not found: 'Void BoDi.IObjectContainer.RegisterInstanceAs(!!0, System.String)'.
```

Does this happen for you?

Thanks,
Sam

**Comment by s-kavanagh** [comment]
Also - I've just tried checking out the code at [this commit](https://github.com/MikeMugu/Zukini/commit/dacf614a46d256d464a5a11c0c0a017193521dfa) (i.e. before the update to Specflow 2.1) and I didn't get this problem.  We only noticed this problem when updating Specflow to V2 or above.

**Comment by MikeMugu** [comment]
Hi Sam,

We have been doing some work on the Zukini code here recently and we
probably need to publish a new nuget package. Let me take a look.

Thanks,

Mike

**Comment by MikeMugu** [comment]
Okay, published latest Nuget package, version 1.1.6. Let me know if you
have any trouble with that one. BTW, we are working on providing API
support libraries soon, hoping to have that done in the next week or so.

Thanks,

Mike

**Comment by s-kavanagh** [comment]
Hi Mike, 
Yes that's sorted it for me.  Many thanks for a quick reply & fix! 
Sam
